src/lib/simulation/gps_trace.cpp

```diff
@@ -0,0 +1,123 @@
+#include "gps_trace.hpp"
+
+#define EARTH_RADIUS 6371000.0
+
+namespace fcpp
+{
```

Comment:
Nella convenzione usata in FCPP, le graffe aperte non vanno a capo ma sulla stessa linea (qui e ovunque sotto accada)